functions/bigquery.js

```diff
@@ -0,0 +1,170 @@
+const functions = require('firebase-functions');
+
+const _ = require('lodash');
+
+const { BigQuery } = require('@google-cloud/bigquery');
+
+const {
+  INBOUND_TABLE,
+  INTAKE_TABLE,
+  REIMBURSEMENTS_TABLE,
+  getAllRecords
+} = require('./airtable');
+
+const bigQueryClient = new BigQuery({
+  projectId: functions.config().bigquery.project_id,
+  credentials: {
+    private_key: functions.config().bigquery.credentials.private_key.split('\\n').join('\n'),
+    client_email: functions.config().bigquery.credentials.client_email,
+  }
+});
+
+const OPS_DATASET_ID = 'operations';
+
+const INBOUND_TYPES = {
+  id: 'STRING',
+  status: 'STRING',
+  intakeTime: 'TIMESTAMP',
+  method: 'STRING',
+};
+
+const INTAKE_TYPES = {
+  id: 'STRING',
+  dateCreated: 'TIMESTAMP',
+  dueDate: 'TIMESTAMP',
+  costCategory: 'INTEGER',
+  householdSize: 'INTEGER',
+  items: 'STRING',
+  neighborhood: 'STRING',
+  status: 'STRING',
+  ticketID: 'STRING',
+  timeline: 'STRING',
+  vulnerability: 'STRING',
+};
+
+const REIMBURSEMENT_TYPES = {
+  id: 'STRING',
+  ticketRecords: 'STRING',
+  totalCost: 'FLOAT',
+  donation: 'FLOAT',
+  netReimbursement: 'FLOAT',
+  // TODO : get rid of this nonesense
```

Comment:
lmao????

Reply:
I want to get rid of that field in the reimbursements table and replace it with a fund manager and status field.